target/common/common.mk

```diff
@@ -24,9 +24,13 @@ TB_SRCS   := $(wildcard ${ROOT}/hw/ip/test/*.sv)
 TB_DIR    := ${ROOT}/target/common/test
 
 VSIM_BENDER   += -t test -t rtl -t simulation -t vsim
-VSIM_SOURCES   = $(shell ${BENDER} script flist ${VSIM_BENDER} | ${SED_SRCS})
```

Comment:
I would keep the `XXX_SOURCES` variables here, since they are used the same also in Occamy. Is there a conflicting reason to move them to the target-specific Makefile?

Reply:
yes, it was needed because the `Makefile` inside the `snitch_cluster` target sets `VSIM_BENDER += -t snitch_cluster`. This additional target flag will not be taken for the bender execution here, and hence the snitch_cluster specific files will be missing from the list.

Reply:
I also like the feedback on which tests passed. This gives you, e.g., when running tests locally, a direct feedback that the tests you wanted to run actually were executed. But as mentioned above, I would not print the individual return codes for the passing tests.

Reply:
That shouldn't actually be an issue for the `VSIM_SOURCES` as defined previously.
https://www.gnu.org/software/make/manual/html_node/Recursive-Assignment.html

I do agree that this is a problem with the `VLT_SOURCES` if we ever decide to extend it in the target directory. This can be corrected by defining it as a recursively expanded variable instead of a simply extended variable.

It seems to work from my tests, I can push this change.

Reply:
Ah nice, I was not aware of that feature. Yes, sure go ahead with it :)